code/modules/mob/living/bloodcrawl.dm

```diff
@@ -136,23 +136,25 @@
 	// but only for a few seconds
 	addtimer(CALLBACK(src, /atom/.proc/remove_atom_colour, TEMPORARY_COLOUR_PRIORITY, newcolor), 6 SECONDS)
 
-/mob/living/proc/phasein(obj/effect/decal/cleanable/B)
-	if(notransform)
-		to_chat(src, span_warning("Finish eating first!"))
-		return FALSE
-	B.visible_message(span_warning("[B] starts to bubble..."))
-	if(!do_after(src, 20, target = B))
-		return
-	if(!B)
```

Comment:
you should keep this (but obv renamed to target) because a blood decal could be cleaned after 2 seconds (even if unlikely)

Reply:
`do_after()` already returns FALSE if the target or the source gets deleted.